app/src/main/java/com/example/mytrainingpal/model/TheMuscleBase.kt

```diff
@@ -63,6 +68,23 @@ abstract class TheMuscleBase : RoomDatabase() {
                                             }
                                         }
 
+                                        fun connectWorkoutEntryWithExercises(
+                                            workoutId: Long,
+                                            exerciseIds: Array<Long>,
+                                        ) {
+                                            for (exerciseId in exerciseIds) {
+                                                workoutEntryExerciseMapDao.insert(
+                                                    WorkoutEntryExerciseMap(
+                                                        workoutId,
+                                                        exerciseId,
+                                                        4,
+                                                        "10,10,10,10",
+                                                        20
+                                                    )
+                                                )
+                                            }
+                                        }
+
```

Comment:
Is this for testing purposes? Otherwise we probably don't want default weorkouts in the database, I guess

Reply:
Jup just for testing purposes. I'd keep them even for the presentation so we can show, that we save past workouts.